deploy/collect-profiles.cronjob.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: collect-profiles
+spec:
+  schedule: "*/15 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: collect-profiles
+          containers:
+          - name: collect-profiles
+            image: quay.io/agreene/collect-profiles:test
+            imagePullPolicy: IfNotPresent
+            command:
+            - bin/collect-profiles
+            args: 
+            - -n
+            - olm
+            - --config-path
+            - /etc/config/
+            - --tls-cert-path 
+            - /etc/pki/tls/certs/
+            - olm-operator-heap-:https://olm-pprof-service:8443/debug/pprof/heap
+            volumeMounts:
+            - mountPath: /etc/config
+              name: config-volume
+            - mountPath: /etc/pki/tls/certs/
+              name: secret-volume
+          volumes:
+          - name: config-volume
+            configMap:
+              name: collect-profiles-config
+          - name: secret-volume
+            secret:
+              secretName: olm-serving-cert
+          restartPolicy: OnFailure
```

```suggestion
          restartPolicy: Never
```

I would rather wait for the next tick than run the risk of a fast restart cycle.

---

Addressed in the latest commit.